policy/release/sbom_cyclonedx/sbom_cyclonedx.rego

```diff
@@ -148,6 +148,63 @@ deny contains result if {
 	result := lib.result_helper_with_term(rego.metadata.chain(), [id, reference.url, reference.type, msg], id)
 }
 
+# METADATA
```

Comment:
We also need corresponding changes in `sbom_spdx.rego` :sob:

Reply:
Cachi2 currently does not produce an SPDX SBOM, so there's not yet anything to latch onto (as in to the externalReference of type distribution in CycloneDX). Holding out on this until the the comment about rule data format is settled.

Reply:
Ack. We can do this in a follow up PR. No need to block this change. I just want to make sure it doesn't get forgotten. Maybe a follow up JIRA issue?

Reply:
I created [EC-1022](https://issues.redhat.com/browse/EC-1022) in your backlog, hope that's okay.